requirements.txt

```diff
@@ -0,0 +1,2 @@
+kaitaistruct==0.10
```

Comment:
See comment above, let's make a proper package and use `pyproject.toml` for dependencie specification